### PR TITLE
[New] Add SCEditor w/ single package.json

### LINKED ISSUE #9296: [Request] Add SCEditor
**Library name:** SCEditor
**Git repository url:** https://github.com/samclarke/SCEditor
**npm package url(optional):** https://www.npmjs.com/package/sceditor
**License(s):** MIT License
**Official homepage:** http://www.sceditor.com/
**Wanna say something? Leave message here:** Please add this library. Thanks!

### PR DESCRIPTION
**Pull request by taichunmin**
close #9296

# Profile of the lib
 * Git repository (required): https://github.com/samclarke/SCEditor
 * Official website (optional, not the repository): http://www.sceditor.com/
 * NPM package url (optional): https://www.npmjs.com/package/sceditor
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 310
   - Count of watchers: 34
   - Count of forks: 115
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 42
   - Downloads in the last month: 77

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Some useless info for CDNJS in package.json should be removed, and we do not prefer to use `"**/*"` under a src folder, actually, there is a folder named `minified` which is better for the base path, btw, we prefer to use npm auto-update, would you please use it instead of git auto-update? Many thanks!

**Comment by taichunmin** [comment]
好的，我有空就改上去。

**Comment by taichunmin** [comment]
Hello, @PeterDaveHello. I have updated the `package.json` of SCEditor. Please review again. Thanks.

File: ajax/libs/sceditor/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "editor",
+    "jquery",
+    "ui",
+    "ecosystem:jquery"
+  ],
+  "filename": "jquery.sceditor.min.js",
+  "npmName": "sceditor",
+  "npmFileMap": [
+    {
+      "basePath": "minified"
```
Comment (PeterDaveHello) [inline ajax/libs/sceditor/package.json]: Missing:
```js
files: [
  "**/*"
]
```

@cdnjs/intern2 let's add the missing test of this part.

**Comment by taichunmin** [comment]
After I add `files: ["**/*"]` to `package`, the `auto-update.js` will result a error.
So I removed `files: ["**/*"]` ant commit.
Is it a bug?

**Comment by taichunmin** [comment]
@pvnr0082t Because there are no `author` and `licence` field in `test/schemata/npm-package.json`, I removed two field.  I will add them to `package.json`.

**Comment by PeterDaveHello** [comment]
@taichunmin sorry about the schemata, I've updated it, feel free to send PRs to make it complete, really appreciate!

About the fileMap part, I'll take care of it, no worry.

File: ajax/libs/sceditor/package.json (hunk 1 above)
Comment (x09326) [inline ajax/libs/sceditor/package.json]: https://github.com/cdnjs/cdnjs/pull/9317

**Pull request merged**